AGGREGATE.md

```diff
@@ -246,18 +241,71 @@ utilize techniques like retries to minimize data loss.
   time. All reports that share a privacy budget key should be sent to the aggregation service at the same time (in any order).
 
 
-* The `shared_info` will be a serialized JSON object. The authenticated data for
-  decryption will consist of this string (encoded as UTF-8) with a constant
-  prefix added for domain separation. The string therefore must be forwarded to
-  the aggregation service unmodified. The reporting origin can parse the string
-  to access the encoded fields.
+* The `shared_info` will be a serialized JSON object. This exact string is used
+  as authenticated data for decryption, see [below](#encrypted-payload). The
+  string therefore must be forwarded to the aggregation service unmodified. The
+  reporting origin can parse the string to access the encoded fields.
 
 
 Note: if [debugging](https://github.com/WICG/conversion-measurement-api/blob/main/EVENT.md#optional-extended-debugging-reports)
 is enabled, additional debug fields will be present in aggregatable reports,
 including the cleartext payloads, to allow downstream systems to verify that
 reports are constructed correctly.
 
+#### Encrypted payload
+The `payload` should be a [CBOR](https://cbor.io) map encrypted via
+[HPKE](https://datatracker.ietf.org/doc/draft-irtf-cfrg-hpke/) and then base64
+encoded. The map will have the following structure:
+
+```jsonc
+// CBOR
+{
+  "operation": "histogram",  // Allows for the service to support other operations in the future
+  "reporting_origin": "https://reporter.example",
```

Comment:
Basically just because it's information the reporting origin would already (trivially) know. That being said, it is conceptually shared information, so went ahead and moved it. (Arguably adds a tiny bit of ‘clutter’ to the plaintext)

Reply:
I agree it's kind of cluttering but I am thinking of shared_info as a thing which gets passed around many layers past the receiving endpoint so it will be good to have some kind of "tag" rather than forcing API consumers to log (or hardcode) what it is, especially if e.g. an ad-tech maintains multiple reporting origins.

Reply:
Sg, the shared_info has it now